### PR TITLE
fix(relay): Add the current directory to paths

### LINKED ISSUE #231: relay is failing on windows
```jsx
Module not found: Error: Can't resolve '__generated__/PrintersListQuery.graphql.ts' in 'app\src\renderer'
```

```
Did you mean './__generated__/PrintersListQuery.graphql.ts'?
Requests that should resolve in the current directory need to start with './'.
Requests that start with a name are treated as module requests and resolve within module directories (node_modules).
If changing the source code is not an option there is also a resolve options called 'preferRelative' which tries to resolve these kind of requests in the current directory too.
```

### PR DESCRIPTION
**Pull request by freiksenet**
Fixes #231

### CODE REVIEW

**Comment by freiksenet** [comment]
@kdy1 Can I please have a review?

File: packages/relay/transform/src/lib.rs [hunk 1]
```diff
@@ -219,6 +221,11 @@ impl<'a> Relay<'a> {
             Ok(real_file_name
                 .parent()
                 .unwrap()
+                .join(if self.config.add_current_directory_to_path {
```
Comment (kdy1) [inline packages/relay/transform/src/lib.rs:224]: I thnk this should be unconditional

Comment (kdy1) [reply packages/relay/transform/src/lib.rs:224]: So please remove the option

**Comment by freiksenet** [comment]
@kdy1  Cool, should be good to go.

**Review by kdy1 — APPROVED** [review]

**Pull request merged**